src/components/Grid/Cell.tsx

```diff
@@ -60,11 +56,27 @@ export const Cell = memo(
     const selectionBorderLeft = rightOfSelectionBorder || rightOfFocus || isFocused;
     const selectionBorderTop = belowSelectionBorder || belowFocus || isFocused;
 
+    const cellRef = useRef<HTMLDivElement>(null);
+
+    useEffect(() => {
+      if (!rowAutoHeight) return;
```

Comment:
Wrap this in curly braces please. Single line `if` statements are an insidious source of bugs and they disguise their intent by not looking like control flow, but like an assignment or mutation.